src/core-layers/arc-layer/arc-layer.js

```diff
@@ -54,9 +54,9 @@ export default class ArcLayer extends Layer {
 
     /* eslint-disable max-len */
     attributeManager.addInstanced({
-      instancePositions: {size: 4, accessor: ['getSourcePosition', 'getTargetPosition'], update: this.calculateInstancePositions},
-      instanceSourceColors: {size: 4, type: GL.UNSIGNED_BYTE, accessor: 'getSourceColor', update: this.calculateInstanceSourceColors},
-      instanceTargetColors: {size: 4, type: GL.UNSIGNED_BYTE, accessor: 'getTargetColor', update: this.calculateInstanceTargetColors}
+      instancePositions: {size: 4, accessor: ['getSourcePosition', 'getTargetPosition'], update: this.calculateInstancePositions, transition: true},
```

Comment:
It seems to me that `transition: true` is the 95% case. Would it make sense to make `transition: true` the default so we only have to set it to false for attributes where we want to prevent it? 

I've been keen on keeping the attribute tables as compact as possible, for max readability/comprehensibility, so I'd want to hear a strong argument for why making `true` the default would be a bad idea.

Reply:
It is still unclear what we'd do to interpolate some attributes (e.g. polygon layer). It is safer to have layers opt in this feature so that existing layers will not be broken.

Reply:
Got it. Sounds like for instanced attributes, the default could be true. But for "custom geometry" attributes, it should not. Would that be too subtle, or would it pretty much cover our use case 100%?

Reply:
Path layer is instanced (one instance per line segment) but not transition enabled (not sure how to deal with changes in vertex count).
`instancePickingColors` is not transition enabled either.

Reply:
OK. Those are pretty special cases, so explicitly overriding for those might be OK.

But, since there are complications, I'll let it rest for now.